thorlcr/master/thgraphmanager.cpp

```diff
@@ -288,6 +288,14 @@ void CJobManager::run()
         dp->qn = queueName.get();
     }
 
+    PROGLOG("verifying mp connection to all slaves");
```

Comment:
The idea is that it helps, by  doing upfront rather then when needed.
This is a one-time thing (not per job), making all connection to each other.